react-example/src/App.jsx

```diff
@@ -6,7 +6,8 @@ import serverMock from './Helpers/Mock/Mock.js'
 import { AppProvider } from './App.Context.jsx'
 import { reducer } from './App.Reducer.js'
 import Router from './Router/Router'
-import './Less/config.less'
+import 'Less/config.less'
+import 'Less/general.less'
```

Comment:
`general.less` file is already being imported in `config.less`, the import is using `reference`, you should remove it. Check the less [import docs](https://lesscss.org/features/#import-atrules-feature-import-options) to understand the reference option